bugsnag-spring/src/main/java/com/bugsnag/BugsnagHandlerExceptionResolver.java

```diff
@@ -0,0 +1,50 @@
+package com.bugsnag;
```

Comment:
The reason for putting everything in the `com.bugsnag` package is so that we can add package-private middleware support to the `bugsnag` submodule for use in `bugsnag-spring` (and any middleware that we might want to support later) but don't want to add to the functionality to the public API, e.g. explicitly reporting unhandled exceptions.

Reply:
After in-person discussion I'm happy with this approach